main/fdm_ru.po

```diff
@@ -1010,7 +1010,7 @@ msgid ""
 "Both %1 and QR code are located inside of Preferences page of %2 you're "
 "going to connect to."
 msgstr ""
-"%1 и QR-код находятся на странице настроек %2, к которому вы собираетесь "
+"%1 и QR-код находятся на странице Настроек %2, к которой вы собираетесь "
```

Comment:
This translation is absolutely wrong.

Reply:
This translation is absolutely correct, though it's not exactly a translation but rather just two minor adjustments in two places. Let's go through them step by step:

1. "Настроек" should be written with an uppercase letter, not a lowercase "настроек". It's simply a matter of capitalizing the word. This standardizes it in line with other languages where this term is also capitalized to distinguish it from other words. Here are screenshots from other languages where the word "Настроек/Preferences" is indeed capitalized.

![1](https://github.com/FreeDownloadManagerTeam/FDM6-localization/assets/139626195/7af46b5a-2290-4b69-8213-77c6e2b24506)
![2](https://github.com/FreeDownloadManagerTeam/FDM6-localization/assets/139626195/c1d25872-bc54-47b7-992a-4de5962ac382)
![3](https://github.com/FreeDownloadManagerTeam/FDM6-localization/assets/139626195/8fbfd037-53d4-4bc4-ad48-02b3da094742)
![4](https://github.com/FreeDownloadManagerTeam/FDM6-localization/assets/139626195/8d2cd96c-a76b-4fca-a944-898406a0473c)

2. The gender of the word "которому" is incorrect. It should be "которой". This can be understood from the context of the entire sentence, assuming you are a native speaker and understand the essence of what is being said. If we were to rely on Google Translate, it often performs poorly in translating sentences with different genders for this word depending on the language. Here's an example:

![001](https://github.com/FreeDownloadManagerTeam/FDM6-localization/assets/139626195/1baad1e1-f228-4ad5-970b-a245708811d0)
![002](https://github.com/FreeDownloadManagerTeam/FDM6-localization/assets/139626195/250762ab-c756-47f8-9d36-33a8cebf7f5d)

However, if you look at how the individual word (которой/которому), which is being discussed, is translated, you'll see that the grammatical gender varies between languages, and the reverse translation of the already translated word yields a different result:

![003](https://github.com/FreeDownloadManagerTeam/FDM6-localization/assets/139626195/6bb5058e-2877-4baa-9754-e558ba79ceeb)

DeepL Translator, on the other hand, handles this better and immediately translates with the correct gender for Russian, unlike Ukrainian where the reverse error occurs:

![004](https://github.com/FreeDownloadManagerTeam/FDM6-localization/assets/139626195/abe4748a-c67a-408c-b0bb-8527a99febe7)

So, in conclusion, if you are not a native speaker, you won't understand what's correct. "К странице настроек..." - "страница" is grammatically feminine, so the question should be "к которой" and not "к которому", because "которому" is grammatically masculine. I hope this clarifies things. Your assertion that "This translation is absolutely wrong" is absolutely absurd for a non-native Russian speaker who doesn't understand the context of the translation as a whole.